Tools/autotest/arducopter.py

```diff
@@ -171,7 +171,7 @@ def loiter(self, holdtime=10, maxaltchange=5, maxdistchange=5):
         self.set_rc(2, 1500)
 
         # wait for copter to slow moving
-        self.wait_groundspeed(0, 2)
+        self.wait_groundspeed(0, 1, maintain_target_time=2)
```

Comment:
```suggestion
        self.wait_groundspeed(0, 2, maintain_target_time=2)
```

Reply:
... unless there was a good reason for that change.

Reply:
1m/s ins't slow moving .... it should even be 0.3 (or even better, get the speed contoler limit ) ... 
Otherwise, it too easy to pass the test... just rc 3 1800, then rc 3 1000, you pass the test wait_groundspeed but have never stop. Same if you are drifting at less that 1m/s ... and on loiter that a big issue!
that why the maintain_target_time is important